app/build.gradle

```diff
@@ -204,8 +204,6 @@ dependencyVerification {
 			'com.github.tony19:apktool-lib:1.4.4-3:apktool-lib-1.4.4-3.jar:bdc4e93b1dd6661698feedacc37d203801076deddd7fb350daf98611637a5cd6',
 			'com.github.tony19:logback-android-classic:1.1.1-6:logback-android-classic-1.1.1-6.jar:0d0939c2cd797ecb0b7d679f1e2314a9889a90a633f2fc5b66f11316cbd6626b',
 			'com.github.tony19:logback-android-core:1.1.1-6:logback-android-core-1.1.1-6.jar:9548fd28988d62a8f67b72fafc67167f91f061001513ae25f53921f2982c8c01',
-			'com.gitlab.opentransitmap.public-transport-enabler:enabler:f540e9748e:enabler-f540e9748e.jar:75aa66b301ce38c7447cba8c982751bf0c789b492ee186f7bc0d47899dfb7c73',
-			'com.gitlab.opentransitmap:public-transport-enabler:f540e9748e:public-transport-enabler-f540e9748e.jar:2b947050d2a6112366aa5a349169bdd1c43584e56d2faf411f9de9617086e58b',
```

Comment:
This should be updated instead of removed. You can get the checksums by running `gradle -q calculateChecksums`

Reply:
Done 👍